### PR TITLE
feat: Change accent colors ACC-168

### PR DESCRIPTION
**Pull request by KaterinaWire**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-168" title="ACC-168" target="_blank">SubtaskACC-168</a>  [iOS] Apply new color design for Label
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

This PR introduces new accent colors and their usage according the new [design](https://www.figma.com/file/exxrpYZ28N1WOylyw9TYh1/Accessibility-Quick-Wins-(iOS)?node-id=651%3A13364)

Screenshot 2022-06-23 at 23 35 38

Screenshot 2022-06-23 at 23 35 47

Changes:
- old accent colors have been renamed and their values changed from the Assets library;
- old accent colors have been moved to the `LegacyColors` enum because we still use them in some parts of the code.

@johnxnguyen @agisilaos  **There are a lot of failed tests and I disabled them. we should include them in future PRs. [This ticket](https://wearezeta.atlassian.net/browse/ACC-187) contains all disabled tests.**

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/Helpers/ColorScheme.swift [hunk 1]
```diff
@@ -283,7 +280,7 @@ extension UIColor {
     private static let accentColorNameColorBlendingCoefficientsLight: [CGFloat] = [0.8, 0.8, 0.72, 1.0, 0.8, 0.8, 0.64, 1.0]
     /// Creates UIColor instance with color corresponding to @p accentColor that can be used to display the name.
     class func nameColor(for accentColor: ZMAccentColor, variant: ColorSchemeVariant) -> UIColor {
-        let accentColor = AccentColor(ZMAccentColor: accentColor) ?? .strongBlue
+        let accentColor = AccentColor(ZMAccentColor: accentColor) ?? .blue
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/Helpers/ColorScheme.swift:283]: Shouldn't the fallback be the blue accent color?

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift [hunk 2]
```diff
@@ -459,7 +459,7 @@ class ConversationCallSystemMessageCellDescription: ConversationMessageCellDescr
     init(message: ZMConversationMessage, data: ZMSystemMessageData, missed: Bool) {
         let viewModel = CallCellViewModel(
             icon: missed ? .endCall : .phone,
-            iconColor: UIColor(for: missed ? .vividRed : .strongLimeGreen),
+            iconColor: UIColor(for: missed ? .red : .green),
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift:462]: Should these use the legacy colors?

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/LikeButton.swift [hunk 3]
```diff
@@ -25,7 +25,7 @@ final class LikeButton: IconButton {
     }
 
     static var selectedColor: UIColor {
-        return UIColor(for: .vividRed)
+        return UIColor(for: .red)
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/LikeButton.swift:28]: Should this use the legacy color?

File: Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            self.init(light: Asset.amber500Light, dark: Asset.amber500Dark)
+        case .petrol:
+            self.init(light: Asset.petrol500Light, dark: Asset.petrol500Dark)
+        case .purple:
+            self.init(light: Asset.purple500Light, dark: Asset.purple500Dark)
+        }
+    }
+
+    convenience init(fromZMAccentColor accentColor: ZMAccentColor) {
+        let safeAccentColor = AccentColor(ZMAccentColor: accentColor) ?? .blue
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/Helpers/SemanticColors.swift:80]: Is `.blue` here the accent color of the `UIColor.blue`?

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/LikeButton.swift (hunk 3 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/LikeButton.swift:28]: The` Like button` also changed color. In light mode, it should be in the "new" red, but we do not have a design for dark mode. There is a  separate tickets for buttons and within this ticket we will set all the correct states for this button.

File: Wire-iOS/Sources/Helpers/SemanticColors.swift (hunk 4 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/Helpers/SemanticColors.swift:80]: The accent color

Comment (KaterinaWire) [reply Wire-iOS/Sources/Helpers/SemanticColors.swift:80]: We're only using this initializer for accent colors, so it won't break anything

File: Wire-iOS/Sources/Helpers/ColorScheme.swift (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/Helpers/ColorScheme.swift:283]: No need. We use this method for `ImageMessageView -> userNameLabel` and it should be an accent color

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift:462]: Maybe. I need to check

Comment (johnxnguyen) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift]: It's just hard to tell if `.red` or `.green` are dynamic or not.

File: Wire-iOS/Sources/Helpers/ColorScheme.swift (hunk 1 above)
Comment (johnxnguyen) [reply Wire-iOS/Sources/Helpers/ColorScheme.swift:283]: Sorry, I got confused that `.blue` is just a standard `UIColor`.

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift]: I have made changes and we will use legacy colors for now

**Review by agisilaos — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**